core/controllers/ingest.py

```diff
@@ -104,8 +105,12 @@ def extract_data(excel_file: FileStorage) -> dict[str, pd.DataFrame]:
             sheet_name=None,  # extract from all sheets
             engine="openpyxl",
         )
-    except ValueError:
+    except ValueError as cannot_read:
+        current_app.logger.error(f"Cannot read the excel file. {cannot_read}")
         return abort(500, "Internal Ingestion Error")
+    except BadZipFile as corrupt_error:
+        current_app.logger.error(f"Tried to read excel but the file was corrupted. {corrupt_error}")
+        return abort(400, "excel_file is corrupted")
```

Comment:
Do we need a test for this? There's currently one for a .txt file, nothing for something that specifically raises ```BadZipFile```.

Reply:
We can add a test but all it will do is mock out the response from pd.read_excel and check that the abort and log occurs..  I suppose it's worth it anyway to ensure that `except` block isn't changed / removed

Reply:
Added tests for both exceptions. Combined them to return the same response and log the same message format as they both mean the file is bad